Made dark theme darker

ByteHamster opened the pull request:
Also, changed background of miniplayer to separate from list entries.

Closes #4065

Before/After:

<img width="250" src="https://user-images.githubusercontent.com/5811634/81549892-c09f0c80-937f-11ea-8d90-581cc85d41b9.png" /> <img width="250" src="https://user-images.githubusercontent.com/5811634/81549873-b977fe80-937f-11ea-8019-9a7261537968.png" />
<img width="250" src="https://user-images.githubusercontent.com/5811634/81549888-be3cb280-937f-11ea-9cee-d1268484631c.png" /> <img width="250" src="https://user-images.githubusercontent.com/5811634/81549882-bbda5880-937f-11ea-8bbe-2b3a1c347bdb.png" />

Pentaphon commented:
The blueish dark theme definitely has better contrast than the old "washed out grey" dark theme, so I approve. 

However, the grey miniplayer in the light theme is sure to upset people because it sticks out too much and appears to be on the "drab" side. Grey is just too dull for a theme color, even for a portion of a theme. I suggest making the miniplayer a very light "earth tone" like a "sand" color or a "cream" color. That combination with white is always more pleasing and natural to look at in inside a home or on some kind of art. 

I suggest #F4EEE1 from this example image. I think that color would be nice for a miniplayer on a white theme. It looks complementary to white, warm and natural instead of stark, cold and drab.  Or we can go with something similar. I just think a very light earth tone always looks classier and more pleasant than a dull gray. 

![2fb861109876d865ea0cbaf56a5729ee](https://user-images.githubusercontent.com/49374058/81558493-a0049180-9342-11ea-8982-d82bcf5bccfd.jpg)

Matth7878 commented:
Black theme looks really nice. I was okay with previous one but the new one is really better, it looks more classy.

keunes commented:
I don't use the white theme personally, but I would agree with @Pentaphon (sorry for the mess-up with assigning this issue to you). I feel it's a bit too dark, losing contrast with the text (esp. feed title) and drawing too much attention to it. Both #EFEEEE and #F4EEE1 I think would work (and both I think are an improvement):
<img alt=AP-miniplayer_ORIGINAL src=https://user-images.githubusercontent.com/5811634/81549873-b977fe80-937f-11ea-8019-9a7261537968.png width=200px> <img alt=AP-miniplayer_EFEEEE src=https://user-images.githubusercontent.com/11229646/81744876-18369880-94a4-11ea-97b8-6c9b0f041a83.png width=200px> <img alt=AP-miniplayer_F4EEE1 src=https://user-images.githubusercontent.com/11229646/81744878-18cf2f00-94a4-11ea-92e9-1a2500d4d2e6.png width=200px>

I would also like to throw in another one here: the colour of the space behind non-square images in the miniplayer (#4071) and lists. When non-squared, in lists images have a 'filling' background-colour. I think it would be nice if this were the case for the mini-player too. Currently non-squared images are only glued to the left side - all other sides are 'free'. This is is especially true for really wide images (like The Guardian's).

This visually odd effect I think would be avoided if we apply a similar background-fill to the mini-player as in the list images. It would make the image vertically centred in its own box, visually. Why I'm noting this feedback here: I guess there should be a relation between the different colours.
<img alt=Screenshot_20200512-234332 width=300px src=https://user-images.githubusercontent.com/11229646/81748855-dc530180-94aa-11ea-9d21-cbeda5cf90bf.png> <img alt=Screenshot_20200512-234332_with-fill width=300px src=https://user-images.githubusercontent.com/11229646/81748857-dceb9800-94aa-11ea-99a7-25daf6897c7b.png>
<img alt=Screenshot_20200512-234410_F4EEE1 width=300px src=https://user-images.githubusercontent.com/11229646/81749736-7d8e8780-94ac-11ea-944d-ed5d3374913a.png> <img alt=Screenshot_20200512-234410_F4EEE1_with-fill width=300px src=https://user-images.githubusercontent.com/11229646/81749738-7e271e00-94ac-11ea-9eb3-e8084f7ee02a.png>
(just thinking the fill in the light theme should be lighter than the mini-player colour, rather than darker as I did here.)

Comparison with different episode cover:
[dark no-fill](https://user-images.githubusercontent.com/11229646/81746404-b4fa3580-94a6-11ea-9f2e-ded791d7c19a.png)
[dark with-fill](https://user-images.githubusercontent.com/11229646/81748036-7023ce00-94a9-11ea-85b7-35aa3526bbec.png)
[F4EEE1 no-fill](https://user-images.githubusercontent.com/11229646/81748283-e58f9e80-94a9-11ea-8b98-8482674191e1.png)
[F4EEE1 with-fill](https://user-images.githubusercontent.com/11229646/81748135-9ba6b880-94a9-11ea-8973-b6992dd4abf6.png)

Pentaphon commented:
> Both #EFEEEE and #F4EEE1 I think would work (and both I think are an improvement):

Now that I look at it in a screenshot, I may have been mistaken. #F4EEE1 does look nice and classy but it might be too different from the rest of the theme because it has a "light sandstone" quality rather than a whitish-look.  #F3ECE7 from that "neutrals" chart also looks good as a nice "eggshell" color to compliment the white theme without looking too different or too gray.  #EFEEEE, as @keunes mentioned, is also an improvement because it has a much less gray tone to it despite being on the gray side. 

One thing is for sure, making the player gray on a light theme looks awful. It looks drab, outdated, and Holo-ish. We should stay away from gray on the light theme altogether and use a very pale earth tone or "off-white cream" color to create a fresh, pleasing, classy color combination.

ByteHamster commented:
> However, the grey miniplayer in the light theme is sure to upset people because it sticks out too much

I don't really like `#F4EEE1` because it looks a bit like yellowed paper that is 100 years old. Changed it to `#EFEEEE` for now but I am open to more suggestions. By the way, the old color (`#e0e0e0`) is the one that is used in Android 10's Settings headers. They are really inconsistent there. Some settings pages have a grey header and some don't.

<img width="100" src="https://user-images.githubusercontent.com/5811634/81808982-df88d480-9520-11ea-9ec4-fffb691520bd.png" />

> the colour of the space behind non-square images in the miniplayer

Done

Updated screenshot:

<img width="250" src="https://user-images.githubusercontent.com/5811634/81808523-1e6a5a80-9520-11ea-84e9-be38efebe3db.png" />

Pentaphon commented:
> I don't really like `#F4EEE1` because it looks a bit like yellowed paper that is 100 years old. Changed it to `#EFEEEE` for now but I am open to more suggestions. By the way, the old color (`#e0e0e0`) is the one that is used in Android 10's Settings headers. They are really inconsistent there. Some settings pages have a grey header and some don't.

Yeah I changed my mind on `#F4EEE1` because it seemed inconsistent with the white. I was just thinking if we had to add a splash (or drizzle of color) it would be a light earth tone, but `#EFEEEE` is more appropriate because it achieves a better level of contrast without looking too grey. I think we might end up sticking with that because according to my quick research, this exact color has also been used before by designers in many official off-white, non-grey logos. 

https://encycolorpedia.com/efeeee

The reason `#e0e0e0` didn't work for me despite also being used by professionals in their logos was that it reminded me of when you would select a white Holo list and it would be about that color and it just made the player look drab, harder to read outdated, and out of place on a white theme. I'm really not sure why Android headers use it. Samsung's OneUI headers, for example avoid using too much grey and rightfully so.
